examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Program.cs

```diff
@@ -2,15 +2,25 @@
 using Azure.Identity;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 
 namespace WebDemoWithEventHub
 {
     public class Program
     {
+        private static IConfigurationRefresher refresher = null;
+
         public static void Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
+            IHostBuilder builder = CreateHostBuilder(args);
+            builder.ConfigureServices((services) =>
+            {
+                services.AddSingleton<IConfigRefresher>(new ConfigRefresher() { Refresher = refresher });
```

Comment:
Is this this the right place? Should this be in the startup.cs?

Reply:
It needs the "refresher" instance which is only available here.

---
In reply to: [427593552](https://github.com/Azure/AppConfiguration/pull/329#discussion_r427593552) [](ancestors = 427593552)

Reply:
I've updated it to get the "Refresher" instance from the IConfiguration.Providers list.